contents/node/nodejs.md

```diff
@@ -31,11 +31,11 @@ Now that we know what Node is, let us look at some benefits it has to offer.
 
 ### Benefit 1: Easy to get Started
```

Comment:
Easy to Get Started

Reply:
you didn't miss it, just merged the latest in 😃